indico/web/client/js/react/components/Combobox.jsx

```diff
@@ -0,0 +1,67 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useRef} from 'react';
+
+import 'indico/custom_elements/ind_combobox';
+import {useNativeEvent} from 'indico/react/hooks';
+import {Translate} from 'indico/react/i18n';
+
+export default function Combobox({options, value, onChange, ...inputProps}) {
+  const uncontrolledInputProps = {...inputProps, defaultValue: value};
+  const inputRef = useRef();
+
+  useNativeEvent(inputRef, 'input', onChange);
+  useNativeEvent(inputRef, 'change', onChange);
+
+  return (
+    <ind-combobox>
+      <input ref={inputRef} {...uncontrolledInputProps} type="text" role="combobox" />
```

Comment:
This needs `autoComplete="off"` to avoid ugly cases such as this one:

![image](https://github.com/indico/indico/assets/179599/63d4709a-a500-4ed0-b892-8eed837cad5c)